openmls/src/group/mls_group/tests_and_kats/utils.rs

```diff
@@ -98,3 +99,67 @@ pub(crate) fn setup_client(
     );
     (credential_with_key, key_package_bundle, signature_keys, pk)
 }
+
+pub(crate) fn setup_alice_bob_group<Provider: OpenMlsProvider>(
+    ciphersuite: Ciphersuite,
+    provider: &Provider,
+) -> (
+    MlsGroup,
+    SignatureKeyPair,
+    MlsGroup,
+    SignatureKeyPair,
+    CredentialWithKey,
+) {
+    let group_aad = b"Alice's test group";
```

Comment:
Does this serve a purpose?

Reply:
Not really, I think. This was already here when this function was still using the CoreGroup, so I just kept it as it was. I can remove it if you'd prefer that.

Reply:
I think we shouldn't carry over orphaned/unused parts of the tests, so it's best to remove it.